tests/containers/setup_env.pm

```diff
@@ -6,9 +6,9 @@ sub run {
    my ($self) = @_;
 
    assert_script_run("mkdir -p /root/data/factory/{iso,hdd,other} /root/data/tests");
-   assert_script_run("docker network create testing");
-   assert_script_run("for i in {1..3}; do docker run --rm -d --network testing -e POSTGRES_PASSWORD=openqa -e POSTGRES_USER=openqa -e POSTGRES_DB=openqa --net-alias=db --name db postgres && break; done", timeout => 600);
-   wait_for_container_log("db", "database system is ready to accept connections", "docker");
+   assert_script_run($self->{cre} . " network create testing");
+   assert_script_run($self->{cre} . " run --rm -d --network testing -e POSTGRES_PASSWORD=openqa -e POSTGRES_USER=openqa -e POSTGRES_DB=openqa --network-alias=db --name db docker.io/library/postgres:latest", timeout => 600);
```

Comment:
seems you are reverting the retrying you added just recently?

Reply:
I can't see it in the diff, but likely this was a timeout I mistakenly added when I didn't know where it was timing out

Reply:
But … you removed the retrying, see the diff please. The `for i in {1..3}; … ` is gone